src/main/webapp/app/exam/manage/exam-management.service.ts

```diff
@@ -52,6 +52,28 @@ export class ExamManagementService {
             .pipe(map((res: EntityResponseType) => ExamManagementService.convertCourseResponseDateFromServer(res)));
     }
 
+    /**
+     * Imports an exam on the server using a PUT request.
+     * @param courseId The course id into which the exam should be imported
+     * @param exam The exam with exercises to import.
+     */
+    import(courseId: number, exam: Exam): Observable<EntityResponseType> {
```

Comment:
For this service, GitHub has a lot of check failures. Not sure if they are valid tho.